index.html

```diff
@@ -149,14 +160,16 @@ <h2 class="text-header">Multi-Post
           <ul><li>CSS</li></ul>
           <ul><li>Javasript</li></ul>
         </div>
-        <button class="project-btn"> <a href="#" type="submit">See my projects</a> </button>
+        <button class="project-btn"> <a href="#">See my projects</a> </button>
         </div>
 
       </div>
     </section>
     <!-- About me -->
-    <section class="work-card ">
-      <div class="about fifth-template template later">
+    <section class="about-work-card ">
+      <div class="about fifth-template template ">
+        <div class="wrapper-1">
+
         <h2 class="abt-header">About my self</h2>
       <p class="about-p">
         Hello I’m a software developer! I can help you build , a product , feature or website. Look through some of  my work and experience! If you like what you see  and have a project you need coded, don’t hesitate to contact me.
```

Comment:
- Kindly make sure that you adjust a bit the font color, font family, and deposition of items (add some space between items eg between icons and let connect button, and arrange them in line  )in your About section to make it stick to the figma design :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/230360704-b3381cc0-a1c7-4c00-a6cd-82ed3f6a8f22.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/230362053-a5803422-a453-4bd0-acf1-05941c3e1926.png)


- Kindly add some space between let connect button and icons for the home page and about page to make your design stick to the figma design as required in the project  :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/230362792-52f8c773-904e-4554-b8f7-602f92fe5346.png)

## Figma design